doc/en/user/source/data/webadmin/layergroups.rst

```diff
@@ -19,9 +23,20 @@ Layer group behaviour can be configured by setting its :guilabel:`mode`. There a
 
 If a layer is included in any non *single* mode group, it will no longer be listed in the flat layer list.  It will still be possible to include the layer in other layer groups. 
 
-.. figure:: img/data_layergroups.png
++------------------------+-------+-------------------+----------------+------------------+
```

Comment:
Not the hugest fan of visual tables like this, as I think they can be cumbersome to edit (realigning columns). I prefer list-tables personally. But the table is definitely appreciated.